common/db_store/db.py

```diff
@@ -136,6 +140,26 @@ def save_data(self, data_store_name: str, data, **kwargs):
             # Log.error(f"Unable to fetch data from data store. Error {e}")
             raise DBError(f"Unable to save data to data store. Error {e}")
 
+    def delete_data(self, data_store_name: str, data, **kwargs):
```

Comment:
same here **kwargs

Reply:
Added statement for kwargs in Doc string.